python/llm/src/ipex_llm/transformers/models/utils.py

```diff
@@ -332,6 +332,9 @@ def use_esimd_sdp(q_len, k_len, head_dim, query_states, attention_mask=None):
     if query_states.shape[0] > 1 and device_name.startswith("Intel(R) Data Center GPU Max"):
         # esimd_sdp not support PVC GPU when batch size > 1 for now
         return False
+    if query_states.shape[0] > 1 and device_name.startswith("Intel(R) Arc(TM) A"):
+        # esimd_sdp not support ARC GPU when batch size > 1 for now
+        return False
```

Comment:
Shall we return false when query_states.shape[0] > 1?

Reply:
> Shall we return false when query_states.shape[0] > 1?

But this works and have performance optimization for bs>1 single A770 inference. I added the limitation now, only using deepspeed and bs>1 on ARC will disable.